cmd/authelia-scripts/docker.go

```diff
@@ -4,8 +4,8 @@ package main
 type Docker struct{}
 
 // Build build a docker image
-func (d *Docker) Build(tag string, target string) error {
-	return CommandWithStdout("docker", "build", "-t", tag, target).Run()
+func (d *Docker) Build(tag string, dockerfile string, target string) error {
```

Comment:
why not parameterizing the command with an architecture instead of the name of a dockerfile? That would probably make more sense for a user of the command. Btw, this arch customization should be a documented flag of the build command. That way users can see it in the usage of the command.

Reply:
I think this ties into my comment further above regarding the fact that I thought these were just helpers for the Travis deployment pipeline.

In the multistage Travis build we are specifying the Architecture and the respective Dockerfile for said Arch.

Reply:
You are right, I misread this part. Dockerfile name is the right level of abstraction in this helper function. My bad. Nothing to check regarding Travis though.